### DISCUSSION TITLE
[question] what is accelerateGeometry in initialize() ?

### DISCUSSION THREAD

**Discussion by mdespriee** [RomanIakovlev/timeshape#73] [Q&A]
Is there any performance gain to expect ?
What's the impact on precision ?

**Comment by RomanIakovlev** [comment]
Yes, it increases the speed of the answer to the 'point in polygon' question, at the expense of increased memory usage (by using a bitmap index). I don't have the actual figures on top of my head, but I remember the speedup was sufficient to introduce this parameter to the API, and the memory usage increase was rather moderate. There's no impact on precision whatsoever.

**Comment by RomanIakovlev** [comment]
I think it's also worth mentioning that accelerating the geometry has some impact on the time it takes to initialize the `TimeZoneEngine`. Again, no exact figures on top of my head, but it probably adds another couple of seconds to initialization.

**Comment by RomanIakovlev** [comment]
I'm going to try the new shiny GitHub Discussions, and use this issue/question as a guinea pig, hope you don't mind @mdespriee. This issue will be converted to a discussion, and could be found on the Discussions tab.
